BUILD.bazel

```diff
@@ -373,6 +373,10 @@ cc_binary(
     name = "lookup_benchmark",
     testonly = 1,
     srcs = ["lookup_benchmark.cc"],
+    data = [
```

Comment:
I don't see this being used anywhere.

Reply:
I added this because in the README there is an example for `lookup_benchmark` which uses the csv file. This `data` entry should make it easier for people who don't know much about blaze to get started and reproduce some of the numbers.

Reply:
Makes sense.